druid/Dockerfile

```diff
@@ -43,13 +43,29 @@ COPY --chown=${STACKABLE_USER_UID}:0 druid/stackable/patches/${PRODUCT} /stackab
 # with a "directory not empty" error on the first builder to finish, as other builders
 # are still working in the cache directory.
 
+# Let's have patchable as a dedicated step, as it fetches the druid sourcecode over the network,
+# thus taking a bit (which is annoying while development)
+RUN /stackable/patchable --images-repo-root=src checkout druid ${PRODUCT} > /tmp/DRUID_SOURCE_DIR
+
 RUN --mount=type=cache,id=maven-${PRODUCT},uid=${STACKABLE_USER_UID},target=/stackable/.m2/repository \
   --mount=type=cache,id=npm-${PRODUCT},uid=${STACKABLE_USER_UID},target=/stackable/.npm \
   --mount=type=cache,id=cache-${PRODUCT},uid=${STACKABLE_USER_UID},target=/stackable/.cache \
   <<EOF
-cd "$(/stackable/patchable --images-repo-root=src checkout druid ${PRODUCT})"
+cd "$(cat /tmp/DRUID_SOURCE_DIR)" || exit 1
+rm /tmp/DRUID_SOURCE_DIR
+
+mvn \
+  --batch-mode \
+  --no-transfer-progress \
+  clean install \
+  -Pdist,stackable-bundle-contrib-exts \
+  -Dmaven.test.skip `# Skip test compilation` \
+  -DskipTests `# Skip test execution` \
+  -Dcheckstyle.skip `# Skip checkstyle checks. We dont care if the code is properly formatted, it just wastes time` \
+  -Dmaven.javadoc.skip=true `# Dont generate javadoc` \
+  -Dmaven.gitcommitid.skip=true \
```

Comment:
@NickLarsenNZ as you added this, can you please add a comment what this is about?

Reply:
```suggestion
  -Dmaven.gitcommitid.skip=true `# The gitcommitid plugin cannot work with git workspaces (ie: patchable)` \
```

Reply:
Done in 004b1dc

Reply:
Ahh thanks!